Task3_Android/app/src/main/java/com/example/task3_android/fragment/ListFragment.kt

```diff
@@ -0,0 +1,178 @@
+package com.example.task3_android.fragment
+
+import android.graphics.Rect
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.appcompat.widget.SearchView
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.task3_android.R
+import com.example.task3_android.adapter.BeerAdapter
+import com.example.task3_android.databinding.FragmentListBinding
+import com.example.task3_android.model.Beer
+import com.example.task3_android.model.PagingData
+import com.example.task3_android.retrofit.BeerService
+import com.google.android.material.snackbar.Snackbar
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+
+class ListFragment : Fragment() {
+    private var _binding: FragmentListBinding? = null
+    private val binding get() = requireNotNull(_binding) {
+        "View was destroyed"
+    }
+
+    private val adapter = BeerAdapter { user ->
+        findNavController().navigate(
+            ListFragmentDirections.toDetails(user.id.toString())
+        )
+    }
+
+    private var isLoading = false
+    private var currentPage = 0
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        return FragmentListBinding.inflate(inflater, container, false)
+            .also { binding ->
+                _binding = binding
+            }
+            .root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        loadBeer()
+
+        with(binding) {
+            toolbar.menu
+                .findItem(R.id.action_search)
+                .let { it.actionView as SearchView }
+                .setOnQueryTextListener(object : SearchView.OnQueryTextListener {
+                    override fun onQueryTextSubmit(query: String): Boolean {
+                        return true
+                    }
+
+                    override fun onQueryTextChange(newText: String): Boolean {
+                        return true
+                    }
+                })
+
+            layoutSwiperefresh.setOnRefreshListener {
+                adapter.submitList(emptyList())
+                currentPage = 0
+                loadBeer {
+                    layoutSwiperefresh.isRefreshing = false
+                }
+            }
+
+            val linearLayoutManager = LinearLayoutManager(
+                view.context, LinearLayoutManager.VERTICAL, false
+            )
```

Comment:
если посмотреть в конструктор, то там такие же дефолные параметры, можно было так 
```suggestion
            val linearLayoutManager = LinearLayoutManager(view.context)
```